lib/util/module-resolver.js

```diff
@@ -35,13 +35,10 @@ const DEFAULT_OPTIONS = {
  *      lookup with the highest priority paths coming first.
  * @constructor
  */
-function ModuleResolver(options) {
-    options = options || {};
-
-    this.options = Object.assign({}, DEFAULT_OPTIONS, options);
-}
-
-ModuleResolver.prototype = {
+class ModuleResolver {
```

Comment:
Also, it looks like the `@constructor` tag is no longer needed as it can be derived automatically by JSDoc